### PR TITLE
Replicated Library Chart v0.3.0

### PR DESCRIPTION
**Pull request by diamonwiggins**
Library chart workshop feedback and image tag bug fix

* Fixing a bug that caused image tags not to be processed correctly if they weren't a string
* Updated readme templates to use `.Chart.Version` instead of hardcoding version
* Updated readme to add more links to `ref` were upstream documentation should be used for a field
* Updated readme with breakglass examples of how to merge and overwrite values as well as add your own templates

### CODE REVIEW

File: charts/wg-easy/charts/replicated-library-0.2.0.tgz
Comment (chris-sanders) [inline charts/wg-easy/charts/replicated-library-0.2.0.tgz:1]: Accidental inclusion back this out, also I'm curious if we need to adjust the .gitignore to help avoid this.

File: charts/wg-easy/sample.yaml
Comment (chris-sanders) [inline charts/wg-easy/sample.yaml:1]: Also accidental inclusion needs to be backed out

**Review by chris-sanders — CHANGES_REQUESTED** [review]
I like the intended changes, just think there are some unintended ones included.

File: charts/wg-easy/charts/replicated-library-0.2.0.tgz
Comment (diamonwiggins) [reply charts/wg-easy/charts/replicated-library-0.2.0.tgz:1]: Fixed

File: charts/wg-easy/sample.yaml
Comment (diamonwiggins) [reply charts/wg-easy/sample.yaml:1]: Fixed

**Review by drohnow — APPROVED** [review]
Good stuff! I really like the break glass doc. 
I wonder if we need something like this (instructions) for each deployment type (deployment, statefulset, daemonset)?

**Review by chris-sanders — APPROVED** [review]
Great looks good now!

**Pull request merged**